### PR TITLE
add: product

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
xem danh sách sản phẩm
[- ticket redmine](https://edu-redmine.sun-asterisk.vn/issues/74312)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image
image
image

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: src/public/css/common.css [hunk 1]
```diff
@@ -1,3 +1,8 @@
 .text-color{
     color: #146d4d;
 }
+
+
+body {
+    font-family: 'Montserrat', sans-serif;
+}
```
Comment (huongnt-2545) [inline src/public/css/common.css]: end file

File: src/service/product.service.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { IGetAllParams } from 'src/interface/interface';
+import { ProductRepository } from '../repository/product.repository';
+
+export class ProductsService {
+  productRepository = new ProductRepository();
```
Comment (huongnt-2545) [inline src/service/product.service.ts]: tạo constructor method e nha

File: src/utils/auth/helper.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    paginationHTML += `<a class="page-link" href="?page=${
+      currentPage - 1
+    }&limit=${CONSTANT.LIMIT_DEFAULT}">&laquo;</a>`;
+  } else {
+    paginationHTML += '<span class="page-link">&laquo;</span>';
+  }
+  paginationHTML += '</li>';
+
+  const startPage =
+    currentPage < 5
```
Comment (huongnt-2545) [inline src/utils/auth/helper.ts]: `5` -> sử dụng hằng số đã define trc đó

File: src/views/layouts/main.hbs [hunk 4]
```diff
@@ -42,4 +46,4 @@
   <script src='/js/home/home.js'></script>
 </body>
 
-</html>
+</html>
```
Comment (huongnt-2545) [inline src/views/layouts/main.hbs]: end file

File: src/views/product/index.hbs [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        <div class="w-100">
+            <div class="list-item d-flex justify-content-end">
+                <div class="row row-cols-4 w-100 g-5 ">
+                    {{#each products}}
+                    <div class="">
+                        <div class="card card-product col height-fluid">
+                            <img src="{{this.image}}" class="card-img-top img-fluid" alt="...">
+                            <div class="card-body">
+                                <p class="card-text title-product">{{this.name}}</p>
+                                <p class="card-text price-product">{{this.price}}</p>
```
Comment (huongnt-2545) [inline src/views/product/index.hbs:22]: giá tiền có đơn vị tiền tệ ko e?

Comment (ghost) [reply src/views/product/index.hbs:22]: dạ em định khi xong base feature sẽ locale VND ạ

**Comment by ghost** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**